internal/xds/translator/translator.go

```diff
@@ -598,7 +598,7 @@ func (t *Translator) processTCPListenerXdsTranslation(
 		patchProxyProtocolFilter(xdsListener, tcpListener.EnableProxyProtocol)
 
 		for _, route := range tcpListener.Routes {
-			if err := processXdsCluster(tCtx, &TCPRouteTranslator{route}, &ExtraArgs{metrics: metrics}); err != nil {
+			if err := processXdsCluster(tCtx, &TCPRouteTranslator{route}, &ExtraArgs{metrics: metrics, circuitBreaker: route.CircuitBreaker}); err != nil {
 				errs = errors.Join(errs, err)
```

Comment:
I think this change is unnecessary. The cluster/proxy protocol config are in the route struct and will be taken from there.

Reply:
you are correct - it tests out ok without passing the circuitbreaker around. [35c75cf](https://github.com/gravitational/gateway/pull/68/commits/35c75cf78a4f6e27e950c8fc969b440336f6f708)